tests/backend/realm/test_realm_stats.py

```diff
@@ -0,0 +1,39 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
+
+import pytest
+from uuid import uuid4
+
+from tests.backend.common import realm_create, realm_stats
+from tests.backend.common import vlob_create, block_create
+
+
+@pytest.mark.trio
+async def test_realm_stats_ok(backend, alice, alice_backend_sock, administration_backend_sock, realm):
+    rep = await realm_stats(alice_backend_sock, realm_id=realm)
+    assert rep == {
+        "status": "ok",
+        "data_size": 0,
+    }
+    # Create new metadata
+    await vlob_create(alice_backend_sock, realm_id=realm, vlob_id=uuid4(), blob=b"1234")
+    rep = await realm_stats(alice_backend_sock, realm_id=realm)
+    assert rep == {
+        "status": "ok",
+        "data_size": 4,
+    }
+
+    # Create new data
+    await block_create(alice_backend_sock, realm_id=realm, block_id=uuid4(), block=b"1234")
+    rep = await realm_stats(alice_backend_sock, realm_id=realm)
+    assert rep == {
+        "status": "ok",
+        "data_size": 8,
+    }
+    await block_create(alice_backend_sock, realm_id=realm, block_id=uuid4(), block=b"1234")
```

Comment:
No need for the remaining of the test: this is cheap to do when using the memory backend, but thinks are much slower in PostgreSQL, so better not waste resources ;-)